src/components/standalone/NeAxiosErrorModal.vue

```diff
@@ -4,6 +4,8 @@ import { computed, ref, watch } from 'vue'
 import { useI18n } from 'vue-i18n'
 import { useNotificationsStore } from '@/stores/standalone/notifications'
 
+//// move to lib after removing ubus references
```

Comment:
I wouldn't move this component in the library, it's too much tailored to the specific use than anything else